docs/data/routes/docs/fundamentals/cli/en.md

```diff
@@ -52,68 +110,130 @@ jss create my-jss-app nextjs --prerender SSG --fetchWith GraphQL
 jss create my-jss-app nextjs --prerender SSR --fetchWith REST
```

Comment:
Is it obvious that `prerender` should not be used with RAV frameworks?

Reply:
In the table it is marked as Next.js.